scripts/dev-release.sh

```diff
@@ -14,7 +14,7 @@
 
 for ORB in src/*/; do
   orbname=$(basename $ORB)
-  if [[ $(git log -1 --format="" --name-only | grep "$orbname") ]]; then
+  if [[ $(git diff $COMMIT_RANGE --name-status | grep "$orbname") ]]; then
```

Comment:
Should we remove this file entirely given https://github.com/CircleCI-Public/circleci-orbs/pull/64/files#diff-1d37e48f9ceff6d8030570cd36286a61L75 ?

Reply:
yeah, probably @ndintenfass